src/pages/workspaces/List.js

```diff
@@ -178,13 +179,17 @@ export const WorkspaceList = () => {
           cellRenderer: ({ rowIndex }) => {
             const { accessLevel, workspace: { workspaceId, namespace, name, attributes: { description } } } = sortedWorkspaces[rowIndex]
             const canView = Utils.canRead(accessLevel)
+            const canAccessWorkspace = () => !canView ? setRequestingAccessWorkspaceId(workspaceId) : undefined
 
             return div({ style: styles.tableCellContainer }, [
               div({ style: styles.tableCellContent }, [
                 h(Link, {
                   style: { color: canView ? undefined : colors.dark(0.7), fontWeight: 600, fontSize: 16, ...Style.noWrapEllipsis },
                   href: canView ? Nav.getLink('workspace-dashboard', { namespace, name }) : undefined,
-                  onClick: !canView ? () => setRequestingAccessWorkspaceId(workspaceId) : undefined,
+                  onClick: () => {
+                    canAccessWorkspace()
+                    !!canView && Ajax().Metrics.captureEvent(Events.workspaceOpen, { workspaceName: name, workspaceNamespace: namespace })
```

Comment:
We already have page view events on workspace dashboard. Do we need this as well?

Reply:
@breilly2 My thought with adding workspace:open was that it might be a better proxy for tracking unique opens, since the dashboard event will trigger if you're switching tabs after you've already opened the workspace. But I could also change tack and just throw the name and namespace on that event instead if that seems better, that's also an option. 

@davidangb Agreed that if we stick with workspace:open I'll need to track down other ways a user can open a workspace, that's a good callout.

Reply:
@breilly2 @davidangb I talked to Amelia and she asked that I add name and namespace tracking to the page:view events on workspaces, which I've done with @zarsky-broad 's help. She said that we may as well leave the other event to track opens from the list, as it might come in handy some time and it's already there. I think that should cover all the bases!

Reply:
I really thought workspace namespace and name were already there. 🤦 🤷 
I'm fine with keeping the new event as well, as long as its name reflects what it is to avoid it from being misinterpreted. Adding details to the Mixpanel lexicon may help with that, and is probably good practice to do anyway.

Reply:
Yeah me too originally… 😬 But it's there now! Yep yep I renamed the new event to workspace:open:from:list (unless you can think of a more concise way to say that) to avoid confusion.